src/components/layout/header/scroll-effects.ts

```
@@ -107,16 +140,48 @@ export class ScrollEffectsManager {
+   */
+  public destroy(): void {
+    this.eventCleanups.forEach(cleanup => cleanup());
+    this.eventCleanups = [];
+    
+    if (this.observer) {
+      this.observer.disconnect();
+      this.observer = null;
+    }
+    
+    // Clean up sentinel element to prevent memory leak
+    if (this.sentinel && this.sentinel.parentNode) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)